juration.js

```diff
@@ -8,6 +8,15 @@
  */
 
 (function() {
+  var setEnglishUnits;
+  var setGermanUnits;
+  var setFrenchUnits;
+
+  if (typeof require !== 'undefined') {
+    var setEnglishUnits = require('./languages/en');
+    var setGermanUnits = require('./languages/de');
```

Comment:
In my last PR, I switched from using just the language ('de') for German to using the full locale ('de-DE').  Can you make the same switch here?

Reply:
We're going to do this at a later time so that the change doesn't merge prior to the change in our application.